backend/app.py

```diff
@@ -1,9 +1,10 @@
 from flask import Flask
 from flask_pymongo import PyMongo
-# from bson.json_util import dumps
+from bson.json_util import dumps
 from bson.objectid import ObjectId
-from flask import jsonify, request
+from flask import jsonify, request, url_for
 from flask_cors import CORS
+from werkzeug.security import generate_password_hash, check_password_hash
 
```

Comment:
if there are some imports that aren't being used, just take them out to make the code cleaner.

same goes for commented out lines of code, instead could include a descriptive comment or just remove entirely.